DataManagementSystem/DB/FileCatalogComponents/FileManagerBase.py

```diff
@@ -1029,7 +1029,7 @@ def getReplicasByMetadata(self, metaDict, path, allStatus, credDict, connection=
     result = self.db.fmeta.findFilesByMetadata(metaDict, path, credDict, extra=True)
     if not result['OK']:
       return result
-    fileIDLFNs = result['Value']
+    fileIDLFNs = result['LFNIDDict']
```

Comment:
Really? Shouldn't it be `result['Value']['LFNIDDict']` ?

Reply:
Yes, it is like that. Returning result in the Value field is just a convention. We return additional information at the same level as Value in many places

Reply:
I am pretty positive we had a lengthy conversation at a BILD, concluding that the form should always be `OK`, `Value/Message`, and that these places were it is not the case should disappear little by little.

Reply:
I agree in general. This is quite a lot of fixes to be done to avoid that. Well, just done for this particular case